services/route_manager/src/main/java/com/futurewei/alcor/route/service/Impl/NeutronRouterServiceImpl.java

```diff
@@ -645,12 +703,18 @@ public InternalRouterInfo constructInternalRouterInfo (String routerId, List<Int
     }
 
     @Override
-    public List<InternalSubnetRoutingTable> constructInternalSubnetRoutingTables(Router router) {
+    public List<InternalSubnetRoutingTable> constructInternalSubnetRoutingTables(Router router) throws DatabasePersistenceException, CanNotFindSubnet, OwnMultipleSubnetRouteTablesException, CacheException, ResourcePersistenceException, ResourceNotFoundException, OwnMultipleVpcRouterException, CanNotFindVpc, CanNotFindRouteTableByOwner {
         if (router == null) {
             return new ArrayList<>();
         }
 
-        List<RouteTable> neutronSubnetRouteTables = router.getNeutronSubnetRouteTables();
+//        List<RouteTable> neutronSubnetRouteTables = router.getNeutronSubnetRouteTables();
+//        if (neutronSubnetRouteTables == null) {
+//            return new ArrayList<>();
+//        }
```

Comment:
@kevin-zhonghao If not needed, please remove it.

Reply:
Sure